src/main/java/version1/gameUtil/screens/RegistrationScreen.java

```diff
@@ -125,7 +127,8 @@ public RegistrationScreen setRegisterButtonText(String registerButtonText) {
     }
 
     public RegistrationScreen setGoToLoginScreenButtonText(String goToLoginScreenButtonText) {
-        this.goToLoginScreenButton = new JButton(goToLoginScreenButtonText);
+        //this.goToLoginScreenButton = new JButton(goToLoginScreenButtonText);
```

Comment:
You can also just remove the commented line all together

Reply:
Fixed!